CHANGELOG.next.asciidoc

```diff
@@ -79,6 +79,10 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Do not rotate log files on startup when interval is configured and rotateonstartup is disabled. {pull}17613[17613]
 - Fix `setup.dashboards.index` setting not working. {pull}17749[17749]
 - Fix Elasticsearch license endpoint URL referenced in error message. {issue}17880[17880] {pull}18030[18030]
+- Fix panic when assigning a key to a `nil` value in an event. {pull}18143[18143]
```

Comment:
changelog didn't make it :(

Reply:
Thanks for catching. Fixed in d4d819117.

While fixing this I noticed that there are duplicate entries for the same PR in the CHANGELOG, even in the original PR to `master`. So I've cleaned those up in this backport. I'll make a separate PR to `master` to clean up the CHANGELOG there.

Reply:
https://github.com/elastic/beats/pull/18233